buildlib/pr/cuda_compatible.yml

```diff
@@ -0,0 +1,114 @@
+          ../contrib/configure-release --with-cuda --prefix=$(Build.Repository.LocalPath)/install
+        displayName: Configure
+
+      - bash: |
+          set -eEx
+          cd pkg-build
+          make -s -j `nproc`
+          make install
+          tar czf $(Build.ArtifactStagingDirectory)/ucx_bin_$(Build.BuildId).tar.gz -C $(Build.Repository.LocalPath)/install .
+        displayName: Build package
+      - task: PublishBuildArtifacts@1
+        inputs:
+          pathToPublish: '$(Build.ArtifactStagingDirectory)'
+          artifactName: ucx
+
+  - job: test_cuda11_2_4
+    dependsOn: ucx_build
+    workspace:
+      clean: all
+    displayName: Test with perftest and ucx_info
+    pool:
+      name: MLNX
+      demands: ucx_gpu_test -equals yes
+
+    strategy:
+      matrix:
+        centos7_cuda11_2:
+          build_container: centos7_cuda11
+          cuda_version: dev/cuda11.2
+        centos8_cuda11_2:
+          build_container: centos8_cuda11
+          cuda_version: dev/cuda11.2
+        ubuntu16_cuda11_2:
+          build_container: ubuntu16_cuda11
+          cuda_version: dev/cuda11.2
+        ubuntu18_cuda11_2:
+          build_container: ubuntu18_cuda11
+          cuda_version: dev/cuda11.2
+        ubuntu20_cuda11_2:
+          build_container: ubuntu20_cuda11
+          cuda_version: dev/cuda11.2
+        centos7_cuda11_4:
+          build_container: centos7_cuda11
+          cuda_version: dev/cuda11.4
+        centos8_cuda11_4:
+          build_container: centos8_cuda11
+          cuda_version: dev/cuda11.4
+        ubuntu16_cuda11_4:
+          build_container: ubuntu16_cuda11
+          cuda_version: dev/cuda11.4
+        ubuntu18_cuda11_4:
+          build_container: ubuntu18_cuda11
+          cuda_version: dev/cuda11.4
+        ubuntu20_cuda11_4:
+          build_container: ubuntu20_cuda11
+          cuda_version: dev/cuda11.4
+
+    container: $[ variables['build_container'] ]
+
+    steps:
+      - checkout: none
+        clean: true
+      - task: DownloadBuildArtifacts@0
+        displayName: 'Download Build Artifacts'
+        inputs:
+          artifactName: ucx
+          downloadPath: $(System.DefaultWorkingDirectory)
+      - bash: |
+          set -eEx
+          tar xzf ucx/ucx_bin_$(Build.BuildId).tar.gz -C $PWD
+          nvidia-smi
+      - bash: |
+          set -eEx
+          . /etc/profile.d/modules.sh
+          module load $(cuda_version)
+          ./bin/ucx_info -u t -epw
+        displayName: Test ucx_info
+        env:
+          LD_LIBRARY_PATH: $(Build.Repository.LocalPath)/lib:$LD_LIBRARY_PATH
+          MODULEPATH: /hpc/local/etc/modulefiles:$MODULEPATH
+      - bash: |
+          set -Eex
+          . /etc/profile.d/modules.sh
+          module load $(cuda_version)
+          port=$((10000 + $(Build.BuildId)))
+          ./bin/ucx_perftest -t tag_bw -p $port &
+          server_pid=$!
+          sleep 5
+          ./bin/ucx_perftest -t tag_bw -p $port 127.0.0.1 2>&1 | tee perf.txt
```

Comment:
need to pass the following params to both client and server:
1. "-m cuda", to really test cuda memory
2. pls also add the following text file as "buildlib/pr/perftest_msg_pow2" and pass "-b buildlib/pr/perftest_msg_pow2" parameter. to test different message sizes

```
      8 -s       8 -n 20000
     16 -s      16 -n 20000
     32 -s      32 -n 20000
     64 -s      64 -n 20000
    128 -s     128 -n 14000
    256 -s     256 -n 7000
    512 -s     512 -n 3000
   1024 -s    1024 -n 2000
   2048 -s    2048 -n 1000
   4096 -s    4096 -n 1000
   8192 -s    8192 -n 800
  16384 -s   16384 -n 400
  32768 -s   32768 -n 200
  65536 -s   65536 -n 100
 131072 -s  131072 -n 50
 262144 -s  262144 -n 25
 524288 -s  524288 -n 12
1048576 -s 1048576 -n 10
2097152 -s 2097152 -n 10
```